tox.ini

```diff
@@ -15,6 +15,7 @@ deps =
 [testenv:yamllint]
 deps =
     ansible
+    yamllint
```

Comment:
same here, do we still need ansible?

Reply:
I tried it without and (maybe because it is being run by bash as the whitelisted external) it does not work. I can have a look at it.

Reply:
How the hell did that get there? I remember having problems without it, but it does not make sense **at all** to me now. Anyway, thankfully, now I tried with this:
```
[testenv:yamllint]
commands = yamllint -c {toxinidir}/.yamllint {toxinidir}/
```
and it Just Works (TM). So that's going to be in the next version.